packages/components/menu/src/menu-item.tsx

```diff
@@ -345,3 +356,6 @@ const CheckIcon: FC = () => (
     />
   </svg>
 )
+
+CheckIcon.displayName = "CheckIcon"
+CheckIcon.__ui__ = "CheckIcon"
```

Comment:
There’s no need to specify displayName for components that don't use forwardRef.

Reply:
@kurakke 
This was also the case with `Button`. Both `displayName` and `__ui__` were unnecessary cases, so I removed them.

Reply:
OK.